### PR TITLE
po: Add Georgian translation

### PR DESCRIPTION
**Pull request by NorwayFun**

### CODE REVIEW

**Comment by solardiz** [comment]
Thank you for the contribution!

Not knowing the language, we (current maintainers of passwdqc) cannot review the translation itself, nor update it further. Do you intend to keep it up to date with changes we might be making to the English messages later?

I think the two commits should be squashed into one. Can you do that, please?

Other than that, I think we should get this merged.
@ldv-alt What do you say?

**Comment by NorwayFun** [comment]
Hello,
For now I just translated Russian one, since I couldn't find a way to
generate pot file. If there is any way to generate it, yes, I'll keep
it up 2 date.
Could you please tell me how to make them into 1 commit?

**Comment by ldv-alt** [comment]
On Fri, Dec 23, 2022 at 12:15:05PM -0800, NorwayFun wrote:
> For now I just translated Russian one, since I couldn't find a way to
> generate pot file. If there is any way to generate it, yes, I'll keep
> it up 2 date.

Yes, there is "make update_po".

> Could you please tell me how to make them into 1 commit?

You can run "git rebase -i @~2" and then turn the last commit into
"fixup".

File: po/ka.po [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+# Temuri Doghonadze <temuri.doghonadze@gmail.com>, 2022.
+
+msgid ""
+msgstr ""
+"Project-Id-Version: passwdqc 2.0.2\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-03-11 20:00+0000\n"
+"PO-Revision-Date: 2022-12-23 20:41+0100\n"
+"Last-Translator: Temuri Doghonadze <temuri.doghonadze@gmail.com>\n"
+"Language-Team: Georgian <(nothing)\n"
```
Comment (ldv-alt) [inline po/ka.po:15]: Sorry, what is `<(nothing)` in the `Language-Team` tag?

**Comment by NorwayFun** [comment]
That is, meaning "no language team URL/email". It's only me translating it
and my email is already mentioned 2 times.

File: po/ka.po [hunk 2]
```diff
@@ -0,0 +1,308 @@
+# A passphrase strength checking and policy enforcement toolset.
+# Copyright (c) 2000-2003,2005,2008,2010,2013,2016 by Solar Designer
+# Copyright (c) 2008,2009 by Dmitry V. Levin
```
Comment (ldv-alt) [inline po/ka.po:3]: The copyright lines are expected to match the copyright lines of the whole project,
the corresponding comment in po/passwdqc.pot says:
```
Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
```

By the way, copyright lines in po/ru.po are outdated, too.

File: po/ka.po [hunk 3]
```diff
@@ -0,0 +1,308 @@
...
+msgid "based on a dictionary word and not a passphrase"
+msgstr "ეყრდნობა ლექსიკონის სიტყვას და არა საკვანძო ფრაზას"
+
+#: passwdqc_check.c:45
+msgid "based on a common sequence of characters and not a passphrase"
+msgstr "ეყრდნობა სიმბოლოების მიღებულ მიმდევრობას და არა საკვანძო ფრაზას"
+
+#: passwdqc_check.c:48
+msgid "based on a word list entry"
+msgstr "დაფუძნებულია სიტყვების სიის  ჩანაწერზე"
```
Comment (ldv-alt) [inline po/ka.po:300]: msgstr contains two consequent white space characters, while msgid doesn't

File: po/ka.po [hunk 4]
```diff
@@ -0,0 +1,308 @@
...
+msgid "Weak password: %s."
+msgstr "სუსტი პაროლი: %s."
+
+#: pam_passwdqc.c:167
+msgid "Sorry, you've mistyped the password that was generated for you."
+msgstr "უკაცრავად, თქვენთვის დაგენერირებული პაროლი არასწორად აკრიფეთ."
+
+#: pam_passwdqc.c:169
+msgid "Sorry, passwords do not match."
+msgstr "პაროლები არ ემთხვევა."
```
Comment (ldv-alt) [inline po/ka.po:252]: Apparently, `Sorry` is present in msgid but is not present in the translated msgstr.

**Comment by NorwayFun** [comment]
Yes, it is so, because to make "Sorry" to go in this sentence it would
be like twice as long and even then it's not making it to look more
formal or polite.

**Pull request closed without merging**